File: src/main/java/ru/mm/surv/codecs/webm/util/stream/TransferEvent.java
package ru.mm.surv.codecs.webm.util.stream;

import ru.mm.surv.codecs.webm.event.EventImpl;

public class TransferEvent extends EventImpl {

    public static final int STREAM_INPUT = 1;
    public static final int STREAM_OUTPUT = 2;

    public TransferEvent(int type) {
        super(type);
    }
}

File: src/main/kotlin/ru/mm/surv/codecs/webm/util/stream/TransferEvent.kt
package ru.mm.surv.codecs.webm.util.stream

import ru.mm.surv.codecs.webm.event.EventImpl

class TransferEvent(type: Int) : EventImpl(type) {
    companion object {
        const val STREAM_INPUT = 1
        const val STREAM_OUTPUT = 2
    }
}